### ISSUE TITLE
Gen 2

### ISSUE THREAD

**Issue by saliapin** [PokemonGoF/PokemonGo-Bot#5913]
So, Gen 2 is out, at long last! The bot has been catching the new 'mons, but of course their movesets are not in the database. Any word on an update? How long do you think we can safely use the bot before the old version stop working? Cheers!

**Comment by saliapin** [comment]
OK, now I'm getting errors that might be caused by the missing movesets:

  File "/Users/xxx/PokemonGo-Bot/pokemongo_bot/inventory.py", line 652, in data_for
    attack_id, cls.__name__))
ValueError: Attack 281 not found in FastAttacks
[2017-02-16 17:16:20] [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)

Can we turns off the routine that checks the attacks?

**Comment by camnomis** [comment]
To be fair there are so many changes that running it now would be pointless, issues with berries etc, new Pokémon and move sets, the API has been depricayed already it seems so I wouldn't advise using the bot.

Although why would you want to bot now, this is when the game is fun again!

**Comment by MerlionRock** [comment]
Exactly! But don't worry, API will updated soon. Hashing service for 0.57.2 is already in service. It's only for PGoAPI to update some of their values.

**Comment by pogarek** [comment]
files :

- data/pokemon.json
- fast_moves.json 
will need to be updated for gen2. please feel free to do it (in your fork or attach it here).

**Comment by MerlionRock** [comment]
If developer wants to help, you can use my fork of PGoAPI here: https://github.com/MerlionRock/pgoapi/tree/develop

Just take note protos are not updated to 0.57 yet

**Comment by MerlionRock** [comment]
for the new item support, here are their values:

	"1101": "Sun Stone",
	"1102": "Kings Rock",
	"1103": "Metal Coat",
	"1104": "Dragon Scale",
	"1105": "Upgrade"

**Comment by pogarek** [comment]
In the past (when 0.53 got released and gen2 got added to pokedex) I've used below to create pokemon.json

- https://github.com/BrunnerLivio/PokemonDataGraber (which takes data from https://fevgames.net/pokedex/)
- a custom script which added data to json file generated by tool above. The data was taken from :  http://www.serebii.net/pokemongo/pokemon/ 

If you know a better , correct sources, with all the data that we need (check the content of data/pokemon.json file), then let me know. I'll write a script , which will generate the new json file.

Or just update the existing files that we have, if these changes are minor.
